[x264] build failure on x86-windows-static when WSL2 is installed

<!-- ⚠ Please use the generated issue template instead to report a port build failure. ⚠ -->
<!-- ⁉ You can find its location at the end of the console output. ⁉ -->

**Host Environment**

- OS: Windows
- Compiler: MSVC 19.34.31942.0

**To Reproduce**

Steps to reproduce the behavior:

`./vcpkg install x264[asm,core,default-features]:x86-windows-static`

**Failure logs**

**Console output:**
```
[cmake] Building x264[asm,core,default-features]:x86-windows-static...
[cmake] warning: -- Using community triplet x86-windows-static. This triplet configuration is not guaranteed to succeed.
[cmake] -- [COMMUNITY] Loading triplet configuration from: C:\vcpkg\triplets\community\x86-windows-static.cmake
[cmake] -- Installing port from location: C:\vcpkg\buildtrees\versioning_\versions\x264\4ff53c90cd7222ae9d3e8dc579c1d198715585c2
[cmake] -- Downloading mirror-x264-baee400fa9ced6f5481a728138fed6e867b0ff7f.tar.gz...
[cmake] -- Extracting source C:/vcpkg/downloads/mirror-x264-baee400fa9ced6f5481a728138fed6e867b0ff7f.tar.gz
[cmake] -- Applying patch uwp-cflags.patch
[cmake] -- Applying patch parallel-install.patch
[cmake] -- Applying patch allow-clang-cl.patch
[cmake] -- Applying patch configure-as.patch
[cmake] -- Using source at C:/vcpkg/buildtrees/x264/src/e867b0ff7f-fb2486f1d0.clean
[cmake] -- Found external ninja('1.11.0').
[cmake] -- Getting CMake variables for x86-windows-static
[cmake] -- Getting CMake variables for x86-windows-static
[cmake] -- Downloading msys-gzip-1.11-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-bash-5.1.008-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-autoconf-2.71-3-any.pkg.tar.zst...
[cmake] -- Downloading msys-diffutils-3.8-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-binutils-2.37-5-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-libtool-2.4.7-3-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-file-5.41-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-zlib-1.2.11-1-x86_64.pkg.tar.xz...
[cmake] -- Downloading msys-libbz2-1.0.8-3-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-coreutils-8.32-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-grep-3.0-2-x86_64.pkg.tar.xz...
[cmake] -- Downloading msys-sed-4.8-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-libpcre-8.45-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-m4-1.4.19-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-automake-wrapper-11-4-any.pkg.tar.zst...
[cmake] -- Downloading msys-gawk-5.1.0-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-mpfr-4.1.0-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-gmp-6.2.1-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-libreadline-8.1.001-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-ncurses-6.2-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-automake1.16-1.16.3-3-any.pkg.tar.zst...
[cmake] -- Downloading msys-perl-5.32.1-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-libcrypt-2.1-3-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-pkg-config-0.29.2-4-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-make-4.3-3-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-findutils-4.8.0-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-libintl-0.21-1-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-libiconv-1.16-2-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-gcc-libs-11.2.0-3-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst...
[cmake] -- Downloading msys-which-2.21-4-x86_64.pkg.tar.zst...
[cmake] -- Applying patch C:/vcpkg/scripts/msys/compile_wrapper_consider_clang-cl.patch
[cmake] -- Using msys root at C:/vcpkg/downloads/tools/msys2/ca907ea16f6f2ba5
[cmake] -- Downloading msys-mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst...
[cmake] -- Downloading msys-mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst...
[cmake] -- Using msys root at C:/vcpkg/downloads/tools/msys2/9a1ec3f33446b195
[cmake] -- Configuring x86-windows-static-dbg
[cmake] CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
[cmake]     Command failed: C:/vcpkg/downloads/tools/msys2/ca907ea16f6f2ba5/usr/bin/bash.exe --noprofile --norc --debug -c "V=1 CPP='cl.exe -E' CC='cl.exe' CC_FOR_BUILD='touch a.out | touch conftest.exe | true' CPP_FOR_BUILD='touch a.out | touch conftest.exe | true' CXX_FOR_BUILD='touch a.out | touch conftest.exe | true' CXX='cl.exe' RC='rc.exe' WINDRES='rc.exe' AR='lib.exe' LD='link.exe -verbose' RANLIB=':' STRIP=':' NM='dumpbin.exe -symbols -headers' DLLTOOL='link.exe -verbose -dll' CCAS=':' AS='C:/vcpkg/downloads/tools/nasm/nasm-2.16.01/nasm.exe' AS='C:/vcpkg/downloads/tools/nasm/nasm-2.16.01/nasm.exe' ./../src/e867b0ff7f-fb2486f1d0.clean/configure --build=x86_64-pc-mingw32 --host=i686-pc-mingw32 \"--disable-lavf\" \"--disable-swscale\" \"--disable-avs\" \"--disable-ffms\" \"--disable-gpac\" \"--disable-lsmash\" \"--disable-bashcompletion\" \"ac_cv_prog_ac_ct_STRIP=:\" \"gl_cv_double_slash_root=yes\" \"ac_cv_func_memmove=yes\" \"--disable-silent-rules\" \"--verbose\" \"--disable-shared\" \"--enable-static\" \"--enable-debug\" \"--disable-cli\" \"--prefix=/C/Users/julia/source/repos/julianxhokaxhiu/FFNx/.build/vcpkg_installed/x86-windows-static/debug\""
[cmake]     Working Directory: C:/vcpkg/buildtrees/x264/x86-windows-static-dbg
[cmake]     Error code: 2
[cmake]     See logs for more information:
[cmake]       C:\vcpkg\buildtrees\x264\config-x86-windows-static-dbg-err.log
[cmake] 
[cmake] Call Stack (most recent call first):
[cmake]   scripts/cmake/vcpkg_configure_make.cmake:812 (vcpkg_execute_required_process)
[cmake]   buildtrees/versioning_/versions/x264/4ff53c90cd7222ae9d3e8dc579c1d198715585c2/portfile.cmake:88 (vcpkg_configure_make)
[cmake]   scripts/ports.cmake:147 (include)
[cmake] 
[cmake] 
[cmake] error: building x264:x86-windows-static failed with: BUILD_FAILED
[cmake] error: Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
[cmake] Then check for known issues at:
[cmake]     https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+x264
[cmake] You can submit a new issue at:
[cmake]     https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[x264]+Build+error
[cmake] Include '[x264] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
[cmake]     vcpkg-tool version: 2023-01-24-8a88d63f241d391772fbde69af9cab96c3c64c75
[cmake]     vcpkg-scripts version: 93895b28e 2023-02-14 (3 hours ago)
[cmake] 
[cmake] 
[cmake] Please use the prefilled template from C:\Users\julia\source\repos\julianxhokaxhiu\FFNx\.build\vcpkg_installed\vcpkg\issue_body.md when reporting your issue.
[cmake] -- Running vcpkg install - failed
[cmake] CMake Error at C:/vcpkg/scripts/buildsystems/vcpkg.cmake:883 (message):
[cmake]   vcpkg install failed.  See logs for more information:
[cmake]   C:\Users\julia\source\repos\julianxhokaxhiu\FFNx\.build\vcpkg-manifest-install.log
[cmake] Call Stack (most recent call first):
[cmake]   C:/Program Files/CMake/share/cmake-3.25/Modules/CMakeDetermineSystem.cmake:124 (include)
[cmake]   CMakeLists.txt:63 (project)
[cmake] 
[cmake] 
[cmake] -- Configuring incomplete, errors occurred!
```

**stdout-x86-windows-static.log**
```
-- Downloading mirror-x264-baee400fa9ced6f5481a728138fed6e867b0ff7f.tar.gz...
-- Extracting source C:/vcpkg/downloads/mirror-x264-baee400fa9ced6f5481a728138fed6e867b0ff7f.tar.gz
-- Applying patch uwp-cflags.patch
-- Applying patch parallel-install.patch
-- Applying patch allow-clang-cl.patch
-- Applying patch configure-as.patch
-- Using source at C:/vcpkg/buildtrees/x264/src/e867b0ff7f-fb2486f1d0.clean
-- Found external ninja('1.11.0').
-- Getting CMake variables for x86-windows-static
-- Getting CMake variables for x86-windows-static
-- Downloading msys-gzip-1.11-1-x86_64.pkg.tar.zst...
-- Downloading msys-bash-5.1.008-1-x86_64.pkg.tar.zst...
-- Downloading msys-autoconf-2.71-3-any.pkg.tar.zst...
-- Downloading msys-diffutils-3.8-2-x86_64.pkg.tar.zst...
-- Downloading msys-binutils-2.37-5-x86_64.pkg.tar.zst...
-- Downloading msys-libtool-2.4.7-3-x86_64.pkg.tar.zst...
-- Downloading msys-file-5.41-2-x86_64.pkg.tar.zst...
-- Downloading msys-zlib-1.2.11-1-x86_64.pkg.tar.xz...
-- Downloading msys-libbz2-1.0.8-3-x86_64.pkg.tar.zst...
-- Downloading msys-coreutils-8.32-2-x86_64.pkg.tar.zst...
-- Downloading msys-grep-3.0-2-x86_64.pkg.tar.xz...
-- Downloading msys-sed-4.8-2-x86_64.pkg.tar.zst...
-- Downloading msys-libpcre-8.45-1-x86_64.pkg.tar.zst...
-- Downloading msys-m4-1.4.19-2-x86_64.pkg.tar.zst...
-- Downloading msys-automake-wrapper-11-4-any.pkg.tar.zst...
-- Downloading msys-gawk-5.1.0-2-x86_64.pkg.tar.zst...
-- Downloading msys-mpfr-4.1.0-1-x86_64.pkg.tar.zst...
-- Downloading msys-gmp-6.2.1-1-x86_64.pkg.tar.zst...
-- Downloading msys-libreadline-8.1.001-1-x86_64.pkg.tar.zst...
-- Downloading msys-ncurses-6.2-2-x86_64.pkg.tar.zst...
-- Downloading msys-automake1.16-1.16.3-3-any.pkg.tar.zst...
-- Downloading msys-perl-5.32.1-2-x86_64.pkg.tar.zst...
-- Downloading msys-libcrypt-2.1-3-x86_64.pkg.tar.zst...
-- Downloading msys-pkg-config-0.29.2-4-x86_64.pkg.tar.zst...
-- Downloading msys-make-4.3-3-x86_64.pkg.tar.zst...
-- Downloading msys-findutils-4.8.0-1-x86_64.pkg.tar.zst...
-- Downloading msys-libintl-0.21-1-x86_64.pkg.tar.zst...
-- Downloading msys-libiconv-1.16-2-x86_64.pkg.tar.zst...
-- Downloading msys-gcc-libs-11.2.0-3-x86_64.pkg.tar.zst...
-- Downloading msys-msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst...
-- Downloading msys-which-2.21-4-x86_64.pkg.tar.zst...
-- Applying patch C:/vcpkg/scripts/msys/compile_wrapper_consider_clang-cl.patch
-- Using msys root at C:/vcpkg/downloads/tools/msys2/ca907ea16f6f2ba5
-- Downloading msys-mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst...
-- Downloading msys-mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst...
-- Using msys root at C:/vcpkg/downloads/tools/msys2/9a1ec3f33446b195
-- Configuring x86-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: C:/vcpkg/downloads/tools/msys2/ca907ea16f6f2ba5/usr/bin/bash.exe --noprofile --norc --debug -c "V=1 CPP='cl.exe -E' CC='cl.exe' CC_FOR_BUILD='touch a.out | touch conftest.exe | true' CPP_FOR_BUILD='touch a.out | touch conftest.exe | true' CXX_FOR_BUILD='touch a.out | touch conftest.exe | true' CXX='cl.exe' RC='rc.exe' WINDRES='rc.exe' AR='lib.exe' LD='link.exe -verbose' RANLIB=':' STRIP=':' NM='dumpbin.exe -symbols -headers' DLLTOOL='link.exe -verbose -dll' CCAS=':' AS='C:/vcpkg/downloads/tools/nasm/nasm-2.16.01/nasm.exe' AS='C:/vcpkg/downloads/tools/nasm/nasm-2.16.01/nasm.exe' ./../src/e867b0ff7f-fb2486f1d0.clean/configure --build=x86_64-pc-mingw32 --host=i686-pc-mingw32 \"--disable-lavf\" \"--disable-swscale\" \"--disable-avs\" \"--disable-ffms\" \"--disable-gpac\" \"--disable-lsmash\" \"--disable-bashcompletion\" \"ac_cv_prog_ac_ct_STRIP=:\" \"gl_cv_double_slash_root=yes\" \"ac_cv_func_memmove=yes\" \"--disable-silent-rules\" \"--verbose\" \"--disable-shared\" \"--enable-static\" \"--enable-debug\" \"--disable-cli\" \"--prefix=/C/Users/julia/source/repos/julianxhokaxhiu/FFNx/.build/vcpkg_installed/x86-windows-static/debug\""
    Working Directory: C:/vcpkg/buildtrees/x264/x86-windows-static-dbg
    Error code: 2
    See logs for more information:
      C:\vcpkg\buildtrees\x264\config-x86-windows-static-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:812 (vcpkg_execute_required_process)
  buildtrees/versioning_/versions/x264/4ff53c90cd7222ae9d3e8dc579c1d198715585c2/portfile.cmake:88 (vcpkg_configure_make)
  scripts/ports.cmake:147 (include)
```

**config-x86-windows-static-dbg-err.log**
```
./../src/e867b0ff7f-fb2486f1d0.clean/configure: line 3: $'\r': command not found
./../src/e867b0ff7f-fb2486f1d0.clean/configure: line 65: syntax error near unexpected token `$'{\r''
./../src/e867b0ff7f-fb2486f1d0.clean/configure: line 65: `log_check() {
'
```

**Additional context**

This is the [`vcpkg.json`](https://github.com/julianxhokaxhiu/FFNx/blob/master/vcpkg.json) manifest.

--- a/scripts/cmake/vcpkg_configure_make.cmake
+++ b/scripts/cmake/vcpkg_configure_make.cmake
@@ -213,32 +213,36 @@ function(vcpkg_configure_make)
         endif()
         cmake_path(CONVERT "$ENV{PATH}" TO_CMAKE_PATH_LIST path_list NORMALIZE)
         cmake_path(CONVERT "$ENV{SystemRoot}" TO_CMAKE_PATH_LIST system_root NORMALIZE)
+        cmake_path(CONVERT "$ENV{LOCALAPPDATA}" TO_CMAKE_PATH_LIST local_app_data NORMALIZE)
         file(REAL_PATH "${system_root}" system_root)
 
         message(DEBUG "path_list:${path_list}") # Just to have --trace-expand output
 
-        set(find_system_dirs 
-                "${system_root}/system32"
-                "${system_root}/System32"
-                "${system_root}/system32/"
-                "${system_root}/System32/")
+        vcpkg_list(SET find_system_dirs 
+            "${system_root}/system32"
+            "${system_root}/System32"
+            "${system_root}/system32/"
+            "${system_root}/System32/"
+            "${local_app_data}/Windows/Apps"
+            "${local_app_data}/Windows/Apps/"
+        )
 
         string(TOUPPER "${find_system_dirs}" find_system_dirs_upper)
 
-        set(index "-1")
-        foreach(system_dir IN LISTS find_system_dirs find_system_dirs_upper)
-            list(FIND path_list "${system_dir}" index)
-            if(NOT index EQUAL "-1")
+        set(index 0)
+        set(appending TRUE)
+        foreach(item IN LISTS path_list)
+            if(item IN_LIST find_system_dirs OR item IN_LIST find_system_dirs_upper)
+                set(appending FALSE)
                 break()
             endif()
+            math(EXPR index "${index} + 1")
         endforeach()
 
-        if(index GREATER_EQUAL "0")
-            vcpkg_list(INSERT path_list "${index}" ${add_to_env} "${MSYS_ROOT}/usr/bin")
-        else()
-            message(WARNING "Unable to find system32 dir in the PATH variable! Appending required msys paths!")
-            vcpkg_list(APPEND path_list ${add_to_env} "${MSYS_ROOT}/usr/bin")
+        if(appending)
+            message(WARNING "Unable to find system dir in the PATH variable! Appending required msys paths!")
         endif()
+        vcpkg_list(INSERT path_list "${index}" ${add_to_env} "${MSYS_ROOT}/usr/bin")
 
         cmake_path(CONVERT "${path_list}" TO_NATIVE_PATH_LIST native_path_list)
         set(ENV{PATH} "${native_path_list}")

--- a/scripts/cmake/vcpkg_configure_make.cmake
+++ b/scripts/cmake/vcpkg_configure_make.cmake
@@ -201,7 +201,52 @@ function(vcpkg_configure_make)
 
     set(configure_env "V=1")
 
-    # macOS - cross-compiling support
+    # Establish a bash environment as expected by autotools.
+    if(CMAKE_HOST_WIN32)
+        list(APPEND msys_require_packages binutils libtool autoconf automake-wrapper automake1.16 m4 which)
+        vcpkg_acquire_msys(MSYS_ROOT PACKAGES ${msys_require_packages} ${arg_ADDITIONAL_MSYS_PACKAGES})
+        set(base_cmd "${MSYS_ROOT}/usr/bin/bash.exe" --noprofile --norc --debug)
+        vcpkg_list(SET add_to_env)
+        if(arg_USE_WRAPPERS AND VCPKG_TARGET_IS_WINDOWS)
+            vcpkg_list(APPEND add_to_env "${SCRIPTS}/buildsystems/make_wrapper") # Other required wrappers are also located there
+            vcpkg_list(APPEND add_to_env "${MSYS_ROOT}/usr/share/automake-1.16")
+        endif()
+        cmake_path(CONVERT "$ENV{PATH}" TO_CMAKE_PATH_LIST path_list NORMALIZE)
+        cmake_path(CONVERT "$ENV{SystemRoot}" TO_CMAKE_PATH_LIST system_root NORMALIZE)
+        file(REAL_PATH "${system_root}" system_root)
+
+        message(DEBUG "path_list:${path_list}") # Just to have --trace-expand output
+
+        set(find_system_dirs 
+                "${system_root}/system32"
+                "${system_root}/System32"
+                "${system_root}/system32/"
+                "${system_root}/System32/")
+
+        string(TOUPPER "${find_system_dirs}" find_system_dirs_upper)
+
+        set(index "-1")
+        foreach(system_dir IN LISTS find_system_dirs find_system_dirs_upper)
+            list(FIND path_list "${system_dir}" index)
+            if(NOT index EQUAL "-1")
+                break()
+            endif()
+        endforeach()
+
+        if(index GREATER_EQUAL "0")
+            vcpkg_list(INSERT path_list "${index}" ${add_to_env} "${MSYS_ROOT}/usr/bin")
+        else()
+            message(WARNING "Unable to find system32 dir in the PATH variable! Appending required msys paths!")
+            vcpkg_list(APPEND path_list ${add_to_env} "${MSYS_ROOT}/usr/bin")
+        endif()
+
+        cmake_path(CONVERT "${path_list}" TO_NATIVE_PATH_LIST native_path_list)
+        set(ENV{PATH} "${native_path_list}")
+    else()
+        find_program(base_cmd bash REQUIRED)
+    endif()
+
+   # macOS - cross-compiling support
     if(VCPKG_TARGET_IS_OSX OR VCPKG_TARGET_IS_IOS)
         if (requires_autoconfig AND NOT arg_BUILD_TRIPLET OR arg_DETERMINE_BUILD_TRIPLET)
             z_vcpkg_determine_autotools_host_arch_mac(BUILD_ARCH) # machine you are building on => --build=
@@ -232,13 +277,9 @@ function(vcpkg_configure_make)
             debug_message("Using make triplet: ${arg_BUILD_TRIPLET}")
         endif()
     endif()
-    
+
     # Pre-processing windows configure requirements
     if (VCPKG_TARGET_IS_WINDOWS)
-        if(CMAKE_HOST_WIN32)
-            list(APPEND msys_require_packages binutils libtool autoconf automake-wrapper automake1.16 m4 which)
-            vcpkg_acquire_msys(MSYS_ROOT PACKAGES ${msys_require_packages} ${arg_ADDITIONAL_MSYS_PACKAGES})
-        endif()
         if (arg_DETERMINE_BUILD_TRIPLET OR NOT arg_BUILD_TRIPLET)
             z_vcpkg_determine_autotools_host_cpu(BUILD_ARCH) # VCPKG_HOST => machine you are building on => --build=
             z_vcpkg_determine_autotools_target_cpu(TARGET_ARCH)
@@ -267,46 +308,6 @@ function(vcpkg_configure_make)
                 string(APPEND arg_BUILD_TRIPLET " --host=${TARGET_ARCH}-unknown-mingw32")
             endif()
             debug_message("Using make triplet: ${arg_BUILD_TRIPLET}")
-        endif()
-        if(CMAKE_HOST_WIN32)
-            vcpkg_list(SET add_to_env)
-            if(arg_USE_WRAPPERS)
-                vcpkg_list(APPEND add_to_env "${SCRIPTS}/buildsystems/make_wrapper") # Other required wrappers are also located there
-                vcpkg_list(APPEND add_to_env "${MSYS_ROOT}/usr/share/automake-1.16")
-            endif()
-            cmake_path(CONVERT "$ENV{PATH}" TO_CMAKE_PATH_LIST path_list NORMALIZE)
-            cmake_path(CONVERT "$ENV{SystemRoot}" TO_CMAKE_PATH_LIST system_root NORMALIZE)
-            file(REAL_PATH "${system_root}" system_root)
-
-            message(DEBUG "path_list:${path_list}") # Just to have --trace-expand output
-
-            set(find_system_dirs 
-                    "${system_root}/system32"
-                    "${system_root}/System32"
-                    "${system_root}/system32/"
-                    "${system_root}/System32/")
-
-            string(TOUPPER "${find_system_dirs}" find_system_dirs_upper)
-
-            set(index "-1")
-            foreach(system_dir IN LISTS find_system_dirs find_system_dirs_upper)
-                list(FIND path_list "${system_dir}" index)
-                if(NOT index EQUAL "-1")
-                    break()
-                endif()
-            endforeach()
-
-            if(index GREATER_EQUAL "0")
-                vcpkg_list(INSERT path_list "${index}" ${add_to_env} "${MSYS_ROOT}/usr/bin")
-            else()
-                message(WARNING "Unable to find system32 dir in the PATH variable! Appending required msys paths!")
-                vcpkg_list(APPEND path_list ${add_to_env} "${MSYS_ROOT}/usr/bin")
-            endif()
-
-            cmake_path(CONVERT "${path_list}" TO_NATIVE_PATH_LIST native_path_list)
-            set(ENV{PATH} "${native_path_list}")
-
-            set(bash_executable "${MSYS_ROOT}/usr/bin/bash.exe")
         endif()
 
         # Remove full filepaths due to spaces and prepend filepaths to PATH (cross-compiling tools are unlikely on path by default)
@@ -517,13 +518,6 @@ function(vcpkg_configure_make)
     endif()
 
     file(RELATIVE_PATH relative_build_path "${CURRENT_BUILDTREES_DIR}" "${arg_SOURCE_PATH}/${arg_PROJECT_SUBPATH}")
-
-    set(base_cmd)
-    if(CMAKE_HOST_WIN32)
-        set(base_cmd ${bash_executable} --noprofile --norc --debug)
-    else()
-        find_program(base_cmd bash REQUIRED)
-    endif()
 
     # Used by CL 
     vcpkg_host_path_list(PREPEND ENV{INCLUDE} "${CURRENT_INSTALLED_DIR}/include")

--- a/scripts/cmake/vcpkg_configure_make.cmake
+++ b/scripts/cmake/vcpkg_configure_make.cmake
@@ -223,8 +223,8 @@ function(vcpkg_configure_make)
             "${system_root}/System32"
             "${system_root}/system32/"
             "${system_root}/System32/"
-            "${local_app_data}/Windows/Apps"
-            "${local_app_data}/Windows/Apps/"
+            "${local_app_data}/Microsoft/WindowsApps"
+            "${local_app_data}/Microsoft/WindowsApps/"
         )
 
         string(TOUPPER "${find_system_dirs}" find_system_dirs_upper)

